doc/manual/manual.md

```diff
@@ -192,10 +200,15 @@ The basic types available are similar to those in many other languages.
 * Strings of textual characters can be represented in the `String` type.
   For example, the value `"example"` has type `String`.
 
-* The "unit" type, written `()`, is essentially a placeholder. It has
-  only one value, also written `()`. Values of type `()` convey no
-  information. We will show in later sections several cases where this
-  is useful.
+* The "unit" type, written `()`, is essentially a placeholder, similar
+  to `void` in languages like C and Java. It has only one value, also
+  written `()`. Values of type `()` convey no information. We will show
+  in later sections several cases where this is useful.
+
+* Functions are given types that indicate what type they consume and
+  what type they produce. For example, the type `Int -> Bool` indicates
```

Comment:
If the intent is to help people who aren't used to this style of type signature, then we might also want to give an example of a multi-argument function, which is likely even less intuitive.